src/styles/output.css

```
@@ -1243,6 +1264,10 @@
     --tw-font-weight: var(--font-weight-semibold);
     font-weight: var(--font-weight-semibold);
   }
+  .tracking-\[0\.2em\] {
+    --tw-tracking: 0.2em;
+    letter-spacing: 0.2em;
```

Stylelint (17.5.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)